contracts/Nexus.sol

```diff
@@ -546,21 +542,6 @@ contract Nexus is INexus, EIP712, BaseAccount, ExecutionHelper, ModuleManager, U
         else revert UnsupportedExecType(execType);
     }
 
-    /// @notice Checks if a module is installed on the smart account.
-    /// @param moduleTypeId The module type ID.
-    /// @param module The module address.
-    /// @param additionalContext Additional context for checking installation.
-    /// @return True if the module is installed, false otherwise.
-    function _isModuleInstalled(uint256 moduleTypeId, address module, bytes calldata additionalContext) private view returns (bool) {
```

Comment:
@Aboudjem remind me again why was this kept in implementation contract?

Reply:
no particular reason, I think we moved it while we were using the `withHook` modifier, but not really sure